meetings/2023-12-13.md

```diff
@@ -0,0 +1,127 @@
+* DO: In the last year with the rise of AP/fediverse in the media with all the issues around twitter and reddit, there's a lot of overlap and alignment. To me the general public is not aware the these protocols have been in existence for well over a decade. We're in a very unique time where these foundations... with newer protocols there's a lot happening in Social Web space, many different approaches, users can have freedom in terms of their data, their social graph and what that can look like... each protocol takes different approach, there can be some adoption and integration that can make more complete approaches as opposed to what has been done separately. we want to connect with everyone involved, make sure we can bring awareness to the projects. 
+* eP: Thank you for the work you do. Often people who work on specs/implementing become very focused on that and sometimes not good outreach, so what you do is important. How big is the audience of wedistribute? And do you work with articles or other media (video, interviews)? 
+* LH: It's not big. We're the largest probably on the fediverse but ??? Mainly articles, there's work on a podcast, and interviews. Even developers don't talk to each so it's a way of getting to know other similar projects. 
+* DO: No set number. We had an article last week about 1000 reads. Depends on the topic. We have accounts on decentralized and centralized platforms. Making an effort to bolster our social media presence. We have started a podcast that we haven't publicly lauched but we have 4-5 episodes with developers, co-authors of AP and some with AT protocol in the works.. we want to extend that to cover Solid as well. We have an informal relationship with Sebastien Rosset. We're also working on some video media utilising PeerTube platform. Publication has been around for some time. 
+* SC: FYI, there was some coverage of dweb/social approaches in https://gitlab.com/bluesky-community1/decentralized-ecosystem that you might find useful, if not already in your radar.
+* SC: Happy to help if you'd like to go over Solid Protocol or other specs: https://solidproject.org/TR/
+
+
+### WIP Implementation Feedback
+
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistence across platforms. 
```

Comment:
```suggestion
* eP: More of a question: been working on establishing mutual relationship between two social agents. Have a working flow based on SAI. Currently relies on SAI sending through a message (sms, email). I want to optimize copy/paste flow so started using [webshare API](https://www.w3.org/TR/web-share/). Wanted to ask if people have experience with [web share target](https://w3c.github.io/web-share-target/) because it's not consistent across platforms. 
```